config/environments/production.rb

```diff
@@ -37,7 +37,7 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
   config.public_file_server.headers = {
-    "Cache-Control" => "public, s-maxage=#{30.days.to_i}, max-age=#{30.days.to_i}"
+    "Cache-Control" => "public, s-maxage=#{30.days.to_i}, max-age=#{3000.days.to_i}"
```

Comment:
I think it's fine! I was checking the other Forems and it's set even higher :D

Reply:
In 8.2 years we may regret setting it so shot 😅

Reply:
ahahhaah